app/src/apdu_handler.c

```diff
@@ -22,14 +22,25 @@
 
 #include "actions.h"
 #include "addr.h"
+#include "apdu_handler_legacy.h"
 #include "app_main.h"
 #include "coin.h"
 #include "crypto.h"
+#include "parser_txdef.h"
 #include "tx.h"
 #include "view.h"
 #include "view_internal.h"
 #include "zxmacros.h"
 
+#undef INS_GET_VERSION
```

Comment:
A comment here to describe the reason why we redefine this would be great. I know that this is because of the backward compatibility feature with the old app. But it would be great to add it.